app/mailers/confirmation_mailer.rb

```diff
@@ -14,4 +14,11 @@ def cancelation_email(order)
     @order.status = 'canceled'
     mail(to: @order.billing_email, subject: 'Cancelation from Gigafood')
   end
+
+  def move_by_bike_email(email, order)
+    @order = order
+    #@url = "http://example.com/movebybike"
+    #@order.status = 'delivery request'
```

Comment:
Commented out code is not cool...

Reply:
i'll remove it.
We need to set the status later